README.md

```diff
@@ -591,6 +591,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Mayank Gupta](https://github.com/mayank-cse)
   - [Mayur Gupta](https://github.com/Mayurmg)
   - [Mayuresh Deolekar](https://github.com/deolekarmayuresh)
+  - [Mazhar Ansari] (https://github.com/MJCoder15)
```

Comment:
There's an extra space here that should be removed. This is to make the link work properly. 🙂
```suggestion
  - [Mazhar Ansari](https://github.com/MJCoder15)
```